pkg/reconciler/dependencybuild/scripts/gradle-build.sh

```diff
@@ -82,6 +82,11 @@ cp -r $(workspaces.source.path)/workspace $(workspaces.source.path)/source
 
 gradle $INIT_SCRIPTS -DAProxDeployUrl=file:$(workspaces.source.path)/artifacts --info --stacktrace -Prelease.version=$(params.ENFORCE_VERSION) "$@"  | tee $(workspaces.source.path)/logs/gradle.log
```

Comment:
Adjust this to output to logs/gradle-online.log just like the ant build script above?